### DISCUSSION TITLE
[DESPERATE] End Effector using simulation for Mujoco

### DISCUSSION THREAD

**Discussion by moses1998lee** [google-deepmind/mujoco#811] [General]
Hi Everyone,
I'm working on a school project with a deadline of 10 days... and Mujoco is alot, so i'm just hoping someone could give me some advice on whether Mujoco would be able to work for this project idea before i dive deep into learning how to use Mujoco. I lack alot of experience so learning stuff like this takes sooooo much time )':

Anyways:
I'm trying to build a reinforcement learning agent that is able to learn how to move its arm (2DOF) to a specific location (object) from looking at a 2D image of the object and how its arm is moving at each time step.

Basically the idea is to get a robot to learn how to tweak its arm movement through 2d image frames and self learn how to pick up an object by due to time limitations, i'll only be focusing on the robot getting its arm to the goal state which is the location of the object. So if you can imagine, its just kind of a human moving and positioning its arm to a mug or something, but we need to do this using reinforcement learning.

Really hope someone could give some advice!

**Comment by JeanElsner** [comment]
You may want to start by looking at the `reacher` domain of the [dm_control suite](https://github.com/deepmind/dm_control/tree/main/dm_control/suite), which basically does what you want to do. However, I think you'd have to add a camera observation as that task uses position-based observations by default.

**Reply by saran-t** [reply]
The `dm_control.suite` comes with a [pixel wrapper](https://github.com/deepmind/dm_control/blob/main/dm_control/suite/wrappers/pixels.py).

**Reply by JeanElsner** [reply]
Thanks, I remembered something like that from the paper, but couldn't find it in the code at first. Well, in that case you're all set @moses1998lee
